setup.py

```diff
@@ -0,0 +1,30 @@
+from sqlalchemy import create_engine
+import requests
+from Models.Reviews import Review
+from sqlalchemy.orm import sessionmaker
+URI = "postgres://xeeqkeipvcflil:fead0f891152e8f657d7ee59ba256aa36c24fed629a0ee765b6a6aac6da2610f@ec2-54-225-236-102.compute-1.amazonaws.com:5432/d7mipabothqsm0"
+engine = create_engine(URI)
+engine.connect()
+Session = sessionmaker(bind=engine)
+session = Session()
```

Comment:
I highly recommend using `flask_sqlalchemy` -- it'll take care of tying database sessions to request sessions for you.